business/health_test.go

```diff
@@ -244,29 +244,32 @@ var (
 	}
 )
 
-func fakeServicesHealthReview() []core_v1.Service {
```

Comment:
Its usage was gone in https://github.com/kiali/kiali/pull/848.
That was 2 months ago. I'll also remove this since it may no longer be needed again.